wikipron/scrape.py

```diff
@@ -1,11 +1,13 @@
 import re
 import unicodedata
-from typing import cast
+from typing import cast, Dict, Any
 
 import pkg_resources
 import requests
 import requests_html
 
+import time
```

Comment:
This is a built-in, not 3rd party, module so it should be on line 2.

Also there should be a blank line before the typing imports (but that's a "drive-by" issue that you didn't cause...it was already like that).